topics/csharp/index.md

```diff
@@ -1,13 +1,16 @@
 ---
-aliases: c-sharp, csharp-code, csharp7, csharp-core, csharp6, csharp-resources
+aliases: c-sharp
 created_by: Anders Hejlsberg
 display_name: C#
+github_url: https://github.com/dotnet/csharplang
 logo: csharp.png
 related: language, dotnet
-released: June 2000
-short_description: C# (C sharp) is an object-oriented programming language by Microsoft.
+released: January 2002
```

Comment:
Looks like the wiki page has 2000 as the initial release. Do you have any sources to this January 2002 date?

Reply:
According to the wiki page, it first **appeared** in 2000, while its first **release** was in 2002.